libraries/Microsoft.Bot.Builder.AI.QnA/QnAPrompts.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// The context of the question that is returned from the QnA Maker API.  Used to return related follow-up prompts.
+    /// Allows a bot to do qna "multi-turn" conversations from one question to another.  So the user can journey through
+    /// a series of filtered/refined questions to get to a final answer.  This requires the QnA knowledge base to have been
+    /// configured with follow-up prompts to individual questions.
+    /// </summary>
+    public class QnAPrompts
+    {
+        /// <summary>
+        /// Gets or sets the index of the prompt, used for ordering.
+        /// </summary>
+        /// <value>
+        /// The display order.
+        /// </value>
+        [JsonProperty("displayOrder")]
+        public int DisplayOrder { get; set; }
+
+        /// <summary>
+        /// Gets or sets the QnADTO returned from the API - is always null.
```

Comment:
I don't believe that the last part of this comment is correct, "is always null"

The [API reference docs here](https://github.com/microsoft/botbuilder-dotnet/pull/1930/files#diff-eb5a1fc8999c819af7697efb827dd1c9R27) indicate that:
* `qna` is a `QnADTO`
* if there's `QnaId`, then `QnADTO` object is ignored

The last bullet then implying that there are instances where it is not null. 

I have emailed the QnA team directly to confirm, however

Reply:
This will be good to know for sure.  I had already looked at the reference docs - but from experience of using the API, it has always passed back a null response.

Reply:
I've added a qna non initial context test.
I've also raised a question on the documentation around this here: https://github.com/MicrosoftDocs/azure-docs/issues/33335